config/routes.rb

```diff
@@ -4,5 +4,9 @@
   get "weekly-menu", to: "menus#weekly_menu"
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
 
-  get "weekly-menu", to: "menus#weekly_menu"
+  # showing all the week menus of the logged in user
+  get "my-weeks", to: "week_menus#my_weeks"
```

Comment:
What about calling it 'my-menu-plans' or something more clear for the user? The method can be still #my_weeks if it's more clear for us, but I think it should **get** some more understandable URL. We'll discuss :-)